BoomerangCommAudio/BoomerangCommAudio/Server.cpp

```diff
@@ -37,10 +39,119 @@ void RunServer(SOCKET& serverSock)
 	SocketInfo->BytesSEND = 0;
 	SocketInfo->PacketsRECV = 0;
 	SocketInfo->DataBuf.len = BUF_SIZE;
+	//SocketInfo->Overlapped.hEvent = makeWSAEvent();
+
+	//
+	/*
+	int	n, ns, bytes_to_read;
+	int	client_len, port, err;
+	SOCKET sd, new_sd;
+	struct	sockaddr_in server, client;
+	WSADATA WSAData;
+	WORD wVersionRequested;
+
+	wVersionRequested = MAKEWORD(2, 2);
+	err = WSAStartup(wVersionRequested, &WSAData);
+	if (err != 0) //No useable DLL
+	{
+		wsprintf(temp, L"DLL not found!\n");
+		Display(temp);
+		exit(1);
+	}
+
+	// Create a stream socket
+	if ((sd = socket(AF_INET, SOCK_STREAM, 0)) == -1)
+	{
+		wsprintf(temp, L"Can't create a socket");
+		Display(temp);
+		exit(1);
+	}
+
+	// Initialize and set up the address structure
+	memset((char *)&server, 0, sizeof(struct sockaddr_in));
+	server.sin_family = AF_INET;
+	server.sin_port = htons(g_port);
+	server.sin_addr.s_addr = htonl(INADDR_ANY); // Accept connections from any client
+
+												// Bind an address to the socket
+	if (bind(sd, (struct sockaddr *)&server, sizeof(server)) == -1)
+	{
+		wsprintf(temp, L"Can't bind name to socket");
+		Display(temp);
+		exit(1);
+	}
+
+	// Listen for connections
+	// queue up to 5 connect requests
+	listen(sd, 5);
+
+	while (TRUE)
+	{
+		client_len = sizeof(client);
+		if ((new_sd = accept(sd, (struct sockaddr *)&client, &client_len)) == -1)
+		{
+			wsprintf(temp, L"Can't accept client\n");
+			Display(temp);
+			exit(1);
+		}
+
+		wsprintf(temp, L" Remote Address:  %s\n", inet_ntoa(client.sin_addr));
+		Display(temp);
+
+		closesocket(new_sd);
+	}
+	closesocket(sd);
+	WSACleanup();
+	*/
+
+	
+	if (!initializeWSA())
+	{
+		wsprintf(temp, L"WSAStartup failed with error");
```

Comment:
This is not necessary. The wrapper function will automatically print error messages.